thorlcr/msort/tsortl.cpp

```diff
@@ -60,8 +60,10 @@ class CREcheck {
 
 #define RECHECK(b) CREcheck checkRE(b)
 
+static constexpr unsigned __int64 transferStreamSignature = 0x0ea614193fb99496; // unique/random pattern to validate against
```

Comment:
comment hasn't been finished.  rogue connections?

Reply:
Have changed comment to:
// unique/random pattern used to validate header is of expected protocol on connect